### ISSUE TITLE
Multiple definitions of age and race/ethnicity

### ISSUE THREAD

**Issue by wkthompson123** [ABCD-STUDY/DEAP#16]
Running DEAP on internal (abcd-deap.ucsd.edu). When I try using age as an independent variable, the first search item that pops up is "interview_age". If I then put this in the model, the analysis crashes. This is because "age" and "interview_age" are the same but included in the model as separate variables. I highly recommend we include only one of these variables ("age") and delete "interview_age" from nda17.Rds.

Same thing for "race_ethnicity", "race.ethnicity", and "race.eth". Let's only include one of these (it should be "race_ethnicity", but with the numbers replaced by the labels).

Finally, related to this, when I click on "race.ethnicity" in the Analyze module, nothing shows up in the display (a barplot should show up).

<img width="937" alt="screen shot 2018-10-12 at 12 57 41 pm" src="https://user-images.githubusercontent.com/44100683/46891358-701ba200-ce1e-11e8-9471-f88689692613.png">
<img width="1396" alt="screen shot 2018-10-12 at 12 58 09 pm" src="https://user-images.githubusercontent.com/44100683/46891399-904b6100-ce1e-11e8-953b-79db529aecdc.png">

**Comment by Mosqidiot** [comment]
Sorting the search result to make sure "age" is on top of the search result.

**Comment by HaukeBartsch** [comment]
How about we add a test for perfect co-linearity of the model variables. If such a case is found we can remove one of the variables. This might happen in several models.

The age variable does not exist on NDA, only on DEAP. 'interview_age' exists on NDA in every single instrument. Don't you think it can confuse people if an NDA variables does not exist in DEAP? 

I thought the race.ethnicity, race_ethnicity and race.ethn variables are all different. Maybe that is not the case? Some have more levels and some are numeric instead of factor variables. The naming does not really give a hint. Maybe that can be improved instead? Code for calculating each variable is on github.com/ABCD-STUDY/analysis-nda17/.

**Comment by wkthompson123** [comment]
Yes we should either delete the extras or rename them to be more descriptive.

Test for perfect collinearity would be good, or even collinearity above a certain level.

I think it's more confusing to have multiple age variables than to keep one or the other.



___________________________

Wesley K. Thompson, Ph.D.
Associate Professor In Residence,
Division of Biostatistics and Halicioğlu Data Science Institute
Department of Family Medicine and Public Health
University of California, San Diego
Associate Director of the Data Analysis and Informatics Core,
Adolescent Brain and Cognitive Development Study
Director of Statistics,
National Consortium on Alcohol & Neurodevelopment in Adolescence
9500 Gilman Drive
La Jolla, CA 92093-0664
email: wkthompson@ucsd.edu
phone: 626-774-1134

________________________________
